tez-runtime-library/src/main/java/org/apache/tez/runtime/library/common/CompositeInputAttemptIdentifier.java

```diff
@@ -50,6 +50,12 @@ public InputAttemptIdentifier expand(int inputIdentifierOffset) {
     return new InputAttemptIdentifier(getInputIdentifier() + inputIdentifierOffset, getAttemptNumber(), getPathComponent(), isShared(), getFetchTypeInfo(), getSpillEventId());
   }
 
+  public boolean include(int thatInputIdentifier, int thatAttemptNumber) {
```

Comment:
nit: this is more like "includes" because the method answers a question like whether this composite identifier includes another one, rather that taking and action

Reply:
I agree that `includes` is more appropriate for the method. I modified the signature of the method as well as its name, and attached a javadoc on top of it.